File: csharp/PaintCalculator/PaintCalculator.cs
namespace PaintCalculator
{
    public static class PaintCalculator
    {
        public static double CalculatePaintRequiredForRectangle(Rectangle rectangle)
        {
            return rectangle.Area() * PaintFactor;
        }

        public static double CalculatePaintRequiredForSquare(Square square)
        {
            return square.Area() * PaintFactor;
        }

        public static double CalculatePaintRequiredForCircle(Circle circle)
        {
            return circle.Area() * PaintFactor;
        }

        public const double PaintFactor = 0.1;
    }

}

File: python/example/PaintCalculator.py
from example.Shapes import Rectangle, Square, Circle

class PaintCalculator:
    PAINT_FACTOR = 0.1

    def calculate_paint_required(self, shape):
        if isinstance(shape, Rectangle):
            return shape.area() * PaintCalculator.PAINT_FACTOR
        elif isinstance(shape, Square):
            return shape.area() * PaintCalculator.PAINT_FACTOR
        elif isinstance(shape, Circle):
            return shape.area() * PaintCalculator.PAINT_FACTOR